proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,207 @@
+   status. Withdrawal from the association shall take effect at the end of the 
+   following calendar month in which the declaration of withdrawal was 
+   submitted.
+4. Expulsion may occur due to serious violations or non-payment of fees over 
+   12 months. Members must be given the opportunity to justify their actions
+   before expulsion. The final decision is done by the General Assembly or 
+   through an internet vote of the ordinary members.
+5. Members leaving the Association or if the Association is liquidated do not 
+   have a right to reimbursement for contributed assets.
+
+## 5. Organs of the Association
+
+- The General Assembly
+- The Board
+
+## 6. General Assembly
+
+1. The General Assembly consists of all ordinary members, each with one vote.
+2. It takes place at least once a year, with invitations sent to all ordinary 
+   and extraordinary members at least six weeks prior.
+3. An extraordinary General Assembly may be called if requested by at least 
+   20 % of ordinary members. 
+4. The General Assembly has a quorum if the invitation was submitted in proper 
+   form.
+5. A 3/4 majority of votes is required to amend the Articles of Association or 
+   decide the liquidation of the Association.
+6. Ordinary members unable to attend may be represented by another member with 
+   written authorization.
+7. Participation and voting in the General Assembly must also be possible via
+   electronic communication. By resolution of the Board, a General Assembly 
+   may be held exclusively by these means.
+
+## 7. Duties of the General Assembly
+
+1. The General Assembly is the highest decision-making organ and is
+   responsible for all duties unless otherwise assigned.
+2. It elects the Board and can deselect board members.
+3. It decides on member expulsion objections.
+4. It receives the Board’s annual report and auditor’s report, and discharges 
+   the Board’s responsibilities.
+5. It decides on amendments to the Articles of Association and liquidation of 
+   the Association.
+6. It presents annual accounts and reports, and appoints auditors.
+7. It decides on purchases, sales, investments, and raising loans.
+
+## 8. The Board
+
+1. The Board consists of the Chairman, the Second Chairman, and the Treasurer 
+   elected for a 3-year term.
+   
+   TODO: Define the role responsibilities. 
+2. The Board’s duties and meeting procedures are stipulated in the by-laws.
+3. The Board decides on all matters unless a General Assembly decision is 
+   required.
+4. Each Board member has the right to represent the Association.
+5. Provisional board members are appointed in case of a vacancy until the next 
+   General Assembly.
+6. The Board can appoint a full-time director to manage current affairs.
+7. The director participates in General Assemblies and Board meetings.
+8. The Board can amend the Articles of Association for formal reasons as 
+   required by authorities.
+
```

Comment:
9. Functions in the organisation do not represent an extra value of power in decision making concerning development. Development decisions have to be democratic, but based on knowledge, research and expertise.

Reply:
I would put in the in purpose of the association that the association should support Mixxx Development, not excise control over it.

Reply:
You need to write the procedure if a boardmember wants to quit / is not anymore able to acts / passes away .... 
+ how + why can boardmembers be fired

Reply:
I have clarified that